kqueen/storages/etcd.py

```diff
@@ -637,12 +638,17 @@ def validate(self):
                 return False, 'Required field {} is None'.format(field)
 
             if field_object.unique and field_object.value:
-                for k, v in self.list(self.namespace).items():
+                if self.is_namespaced():
```

Comment:
btw what do u think about method naming and method itself
imo at least we should rename it as (is_global) or smth similar
coz global -namespace is namespace too, so imo this method is weird

Reply:
May be we should, but not in scope of this commit